apps/ausweis/src/features/onboarding/screens/id-card-scan.tsx

```diff
@@ -11,19 +19,64 @@ interface OnboardingIdCardScanProps {
   onCancel: () => void
 }
 
+// @TimoGlastra
```

Comment:
@TimoGlastra Could you maybe do this? And maybe also the biometrics request at the new `id-card-verify` screen.